textext/base.py

```diff
@@ -599,27 +599,19 @@ def typ_to_any(self, typst_command, typst_text, preamble_file, file_type):
         """
 
         with logger.debug("Converting .typ to .{0}".format(file_type)):
-            # # Read preamble
-            # preamble_file = os.path.abspath(preamble_file)
-            # preamble = ""
-            #
-            # if os.path.isfile(preamble_file):
-            #     with open(preamble_file, 'r') as f:
-            #         preamble += f.read()
-            #
-            # # Add default document class to preamble if necessary
-            # if not _contains_document_class(preamble):
-            #     preamble = self.DEFAULT_DOCUMENT_CLASS + preamble
-            #
-            # # Options pass to LaTeX-related commands
-            #
-            # texwrapper = self.DOCUMENT_TEMPLATE % (preamble, latex_text)
-
+            # Read preamble
+            preamble_file = os.path.abspath(preamble_file)
+            preamble = ""
+            
+            if os.path.isfile(preamble_file):
+                with open(preamble_file, 'r') as f:
+                    preamble += f.read()
+            
```

Comment:
As far as I do see this breaks the option to not having a document class in the latex preamble. This is a feature some users have so we introduced it.

Reply:
This is in `typ_to_any` so I figured it'd only affect typst side?